.github/dependabot.yml

```diff
@@ -0,0 +1,40 @@
+# Basic dependabot.yml file with minimum configuration for two package managers
+
+version: 2
+updates:
+  # Enable version updates for python
+  - package-ecosystem: "pip"
+    # Look for a `requirements` in the `root` directory
+    directory: ".github/scripts/"
```

Comment:
iiuc according to https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file directory should be the directory containing the requirements file, not .github/scripts/

Reply:
True, but for this project seems to be the only place with traceble requirements...
If we refactor the `setup.py` and
1. move listed requirements to own file or
2. to `pyproject.toml`

I can do it if you want...

Reply:
what do you think @dbieber ?

Reply:
v0.6.0 will be the last version of Python Fire to support Python 2.
After that, I intend to switch to pyproject.toml.